pkg/runner/flags_test.go

```diff
@@ -85,6 +82,41 @@ func TestParseFlags(t *testing.T) {
 				return c
 			},
 		},
+		{
```

Comment:
I think these tests should be in pkg/tenancy/flags_test.go

Reply:
The test approach here is pretty good and keeps all flags related things here. Plus, the way params are expected here vs in tenancy/flags.go makes it more preferable to write those tests here.

Reply:
refer to `fullyParse`